apollo-adminservice/src/test/resources/application.properties

```diff
@@ -3,4 +3,4 @@ spring.jpa.hibernate.naming.physical-strategy=org.hibernate.boot.model.naming.Ph
 spring.jpa.properties.hibernate.show_sql=false
 spring.h2.console.enabled = true
 spring.h2.console.settings.web-allow-others=true
-
+spring.main.allow-bean-definition-overriding=true
```

Comment:
Are there some tests exist bean name confliction?

Reply:
The default value of `spring.main.allow-bean-definition-overriding` was changed to false [since spring boot 2.1](https://github.com/spring-projects/spring-boot/wiki/Spring-Boot-2.1-Release-Notes#bean-overriding), and it looks like some beans would be created 2 times in test mode, so we need to set it explicitly. I think I may need to find out why they are created 2 times...

Reply:
There are duplicated beans defined, e,g. `com.ctrip.framework.apollo.SkipAuthorizationConfiguration`